packages/loot-core/src/server/importers/ynab5.ts

```diff
@@ -136,8 +136,12 @@ async function importTransactions(
   const startingBalanceCatId = categories.find(
     cat => cat.name === 'Starting Balances',
   ).id; //better way to do it?
+
   const startingPayeeYNAB = data.payees.find(
-    payee => payee.name === 'Starting Balance',
+    payee =>
+      payee.name.localeCompare('Starting Balance', undefined, {
```

Comment:
Can we create a `equalsIgnoreCase` function here? And we also do an ignore case equality check on `Starting Balances` category just above this line?

Reply:
Done this, and also applied to to other string literals in the file